testing/browser/tests/test-SFNT.js

```diff
@@ -259,6 +259,7 @@ function testSFNT(SFNT, isTTF) { try {
         assertNotEqual(loca, undefined, `loca EXISTS`);
         const glyf = tables.glyf;
         assertNotEqual(glyf, undefined, `glyf EXISTS`);
+        assertEqual(!!loca.x2, tables.head.indexToLocFormat === 0, `Correct interpretation of indexToLocFormat`);
```

Comment:
Note that we shouldn't need the `!!` here, x2 is either true for format 0, or undefined for format 1.

Reply:
I could change it to `assertEqual(!loca.x2, tables.head.indexToLocFormat === 1). The current implementation of assert functions fail when you pass undefined to them. Both the `equal` and `arrayEqual` functions should use `Array.isArray()` otherwise they throw "undefined is not an object" errors.

Reply:
I see. Inconvenient, but good to know. Let's keep that `!!` in that case.